nbbuild/antsrc/org/netbeans/nbbuild/extlibs/ignored-overlaps

```diff
@@ -45,14 +45,19 @@ nbbuild/external/json-simple-1.1.1.jar ide/libs.json_simple/external/json-simple
 webcommon/javascript2.jade/external/jflex-1.4.3.jar webcommon/javascript2.lexer/external/jflex-1.4.3.jar
 
 # javax.annotation-api is used by multiple modules.
-enterprise/websvc.restlib/external/javax.annotation-api-1.2.jar java/maven.indexer/external/javax.annotation-api-1.2.jar
 enterprise/javaee.api/external/javax.annotation-api-1.2.jar java/maven.indexer/external/javax.annotation-api-1.2.jar
```

Comment:
javax.annotation-api-1.2 upgrade to jakarta.annotation-api:1.3.5

Reply:
I upgraded the javax.annotation-api only inside the `enterprise.websvc` module. In [PR-1809](https://github.com/apache/netbeans/pull/1809) Matthias Bläsing stated that I should only use the versions that belong to each release of JavaEE. I think in this case I should create another PR to upgrade the version of javax.annotation-api for the modules `enterprise/javaee8.api` to javax.annotation-api:1.3.2 and `java/maven.indexer` to jakarta.annotation-api:1.3.5. I hope I understood your comment.